CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## x.x.x x-x-x
+
+### STPPaymentOptionsViewController and STPPaymentContext
```

Comment:
We try to make these names of modules.

I'd make this ## Payments and make the line read 
* [Fixed] An issue where the correct card brand was not being displayed for card brand choice in STPPaymentOptionsViewController and STPPaymentContext